release-tools/mkrelease.pl

```diff
@@ -385,6 +396,51 @@ sub print_branch_info {
 
    The OpenSSL Project Team.
 
+EOF
+
+    } else {
+        ###### PREMIUM RELEASE TEMPLATE ######
+
+        # Using $avers so its length is similar to a real version
+        # length so it's easier to make the announcement look pretty.
+        my $avers = $expected_version;
+
+        my $title = "OpenSSL version $avers released";
+        my $underline = "=" x length($title);
+        print OUT <<EOF;
+
+   $title
+   $underline
+
+   OpenSSL - The Open Source toolkit for SSL/TLS
+   https://www.openssl.org/
+
+   The OpenSSL project team is pleased to announce the release of
+   version $avers of our open source toolkit for SSL/TLS.
+
+   OpenSSL $avers is available for download via SFTP from the
+   following location:
+
+     * Server: ftp.openssl.org
+
+   Accessing it requires that you have registered an SSH key with us.
```

Comment:
Are we ready to say this yet? It probably needs a bit more discussion before we go down this route. If nothing else there needs to be some planning around communicating this change and supporting customers to set this up. It may take a while before everyone is transitioned to the new approach. It may be that we have both the old username/password way and the new ssh key way setup simultaneously for a while.

Reply:
Okie, I've changed it to require a premium password to be given if it's a premium version.